### ISSUE TITLE
Readme suggest

### ISSUE THREAD

**Issue by wolfbolin** [openGemini/opengemini-client-go#133]
I hope to modify README from the perspective of the user.
Can we add describe like this?

Install
```golang
go get github.com/openGemini/opengemini-client-go/opengemini
```
Usage
```golang
import "github.com/openGemini/opengemini-client-go/opengemini"

func main(){
    config := &opengemini.Config{}
}
```

A good description makes it look different from a local testing program.

**Comment by shoothzj** [comment]
I think wo can add go get method make people easier to copy. But dot import seems have their meanings.
https://github.com/openGemini/opengemini-client-go/blob/e15a322d7b9b723bcd406921aaae89964c144829/README.md?plain=1#L25

**Comment by wolfbolin** [comment]
In many code specifications. It is not recommended to use dots to ignore package names unless absolutely necessary.
The dots in the example appear to be there only to avoid overly long package names `opengemini`
To avoid user confusion about the origin of the class, I recommend using the full package name in the example

**Comment by shoothzj** [comment]
I agree not to use dot import. @Chenxulin97 Do you have time to fix this?

**Comment by Chenxulin97** [comment]
@shoothzj  ok I'll fix it later

**Comment by shoothzj** [comment]
`go get` is not added, and dot import notice is not removed

**Comment by shoothzj** [comment]
closed by #152
